seaworthy/fixtures.py

```diff
@@ -0,0 +1,42 @@
+from seaworthy.definitions import ContainerDefinition
+from seaworthy.containers.postgresql import PostgreSQLContainer
+from seaworthy.utils import output_lines
+
+
+class DjangoContainer(ContainerDefinition):
+    WAIT_PATTERNS = (r"Booting worker",)
+
+    def __init__(self, name, db_url,
+                 image="malaria"):
```

Comment:
Think this `__init__()` can be on one line?

Reply:
yes it can